devtools/common/src/main/resources/templates/basic-rest/java/build.gradle-template.ftl

```diff
@@ -26,6 +26,9 @@ dependencies {
 
     testImplementation 'io.quarkus:quarkus-junit5'
     testImplementation 'io.rest-assured:rest-assured'
+
+    nativeTestImplementation 'io.quarkus:quarkus-junit5'
```

Comment:
Can we rename to `testNativeImplementation`? Also rename other occurrences to `testNative` instead of `nativeTest`?

Reply:
Yeah, that makes sense.

Reply:
Fixed